README.md

```diff
@@ -113,244 +112,128 @@ bash test/test_cleanup.sh
-to prepare indexes and other contingent resources that will be used in later
-steps.
-
-#### _MAP_
-
-The second subworkflow, **_MAP_**, aligns the user-provided short read smallRNA-seq
-libraries against the references generated with the **_PREPARE_** subworkflow. For
-increased fidelity it uses two separate aligning tools, [Segemehl][segemehl]
-and our in-house tool [Oligomap][oligomap]. In both cases, reads are aligned
-separately to the genome and the transcriptome. Afterwards, alignments are
-merged in a way that only the best alignment (or alignments) of each read are
-kept.
-
-#### _QUANTIFY_
-
-The third and final subworkflow, **_QUANTIFY_**, quantifies miRNA expression by
-intersecting the alignments from the **_MAP_** subworkflow with the annotations
-generated in the **_PREPARE_** subworkflow. Intersections are computed with
-[`bedtools`][bedtools] for one or multiple of mature, primary transcripts and
-isomiRs. Reads consistent with each miRNA are counted and tabulated.
-
-
-The schema below is a visual representation of
-an example run of the whole workflow:
-
-> ![rule-graph][rule-graph]
-
-
-### Running the workflow
-
 Assuming that you are currently inside the repository's root directory, 
-create a directory from which you will run your workflow and
-name it whatever you want e.g., `MY_ANALYSIS` and 
-head to it.
+create a directory from which you will run your workflow and name it whatever 
+you want e.g., `MY_ANALYSIS` and head to it.
 
 ```bash
 mkdir MY_ANALYSIS
 cd MY_ANALYSIS
 ```
-Now within the `MY_ANALYSIS/` directory, 
-create a new directory to store you library; 
-name it as you like, e.g. `my_lib` and move there.    
+Place your library file(s) here. In addition, create the sample table 
+`sample_table.csv` and fill it up. The `sample_table.csv` is a tab-separated  
+file, with one row per library, that must have the following columns:  
 
-```bash
-mkdir my_lib
-cd my_lib
-```
-Place your library file(s) here. 
-In addition, create a sample table. 
-Fill it with the correct entries. 
-You can look at the `test/test_files/sample_table.csv` 
-to get an idea of what this file must look like.
+- `sample`. This column contains the library name.  
+- `sample_file`. In this column, you must provide the path to the library file.
+The path must be relative to the working directory.  
+- `adapter`.  This field must contain the adapter sequence in capital letters.  
+- `format`. In this field you mast state the library format. It can either be 
+`fa` if providing a FASTA file or `fastq` if the library is a FASTQ file.  
+
+You can look at the `test/test_files/sample_table.csv` to know what this file 
+must look like, or use it as a template.
 
 ```bash
 touch sample_table.csv
 ```
+In this same directory, add the following required input files:  
+
+- The gzipped reference genome in FASTA format.  
+- The gzipped gene annotation file in GTF format.  
+- The unzipped microRNA annotation file in GTF format.  
```

Comment:
Why "The". I don't think you mention them before. Also you could expand the description a bit. Add a section "Prepare genome resources" and then have a little paragraph for each one, including a link to Ensembl or miRBase.

Reply:
Should I explain in this little paragraph how/for what will the file be used? I do not quite know what to write otherwise besides where to download the file. This is what I have rn: 

***

There are 4 files you must provide: 

- A gzipped reference genome in FASTA format. You can download it from [Ensembl][ensembl].  
- A gzipped gene annotation file in GTF format. You must provide the file with any filtering or modification desired already done. You can download it from the [Ensembl][ensembl] web page.
- An unzipped microRNA annotation file in GTF format. You can download this file from the [miRBase][mirbase] portal.
- A tab-separated mapping table between UCSC and Ensembl. You can find this tab-separated file for different organisms [here][chrMap].  

> **Note:** We expect the genome and gene annotations to be formatted according
> the style used by Ensembl. Other formats are very likely to lead to problems.
> The miRNA annotation file is expected to originate from miRBase, or follow 
> their exact layout.